plugins/woocommerce-blocks/assets/js/shared/stores/product-type-template-state/index.ts

```diff
@@ -130,12 +130,12 @@ const reducer = ( state: StoreState = DEFAULT_STATE, action: Actions ) => {
 	}
 };
 
-export const store = createReduxStore( STORE_NAME, {
```

Comment:
All stores registered by core use the `store` constant name. For instance [here](https://github.com/WordPress/gutenberg/blob/a2daa2fc0874e6af82d0607c65b324d873c9e4ff/packages/preferences/src/store/index.js#L21).

We can use anything we choose, but it was intentional to follow the core coding convention. Just FYI.

Indeed you saw how it imports the stores, for instance:

```es6
import { store as noticesStore } from '@wordpress/notices';
```

...which is another convention :-)

Reply:
Aaah, good point. I don't have a strong opinion. I think in WooCommerce we are a bit more liberal with that. In fact, I saw several different patterns in our codebase. :sweat: I'm fine keeping this aligned with core, so I will undo these changes.

Reply:
Done in 3fe15e8a1b82de67e470235be04e0805265660e2. Thanks for the feedback, @retrofox!